packages/appkit/src/universal-adapter/client.ts

```diff
@@ -346,6 +363,31 @@ export class UniversalAdapterClient {
   }
 
   // -- Private -----------------------------------------------------------------
+  private async syncBalance(address: `0x${string}`, caipNetwork: CaipNetwork) {
+    const isExistingNetwork = this.appKit
+      ?.getCaipNetworks(caipNetwork.chainNamespace)
+      .find(network => network.id === caipNetwork.id)
+
+    // How to fetch balance on non-evm networks?
```

Comment:
does Blockchain API not support non-evm? It should?

Reply:
depends on the chain, currently evm solana polkadot and I think btc since yesterday